src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -514,45 +568,63 @@ class PackageDetailTab extends mixin(StoreMixin) {
             />
           }
         />
-        <div className="container">
-          <div className="media-object-spacing-wrapper media-object-spacing-wide media-object-offset">
-            <div className="media-object media-object-align-top media-object-wrap">
-              <div className="media-object-item">
-                <div className="icon icon-huge icon-image-container icon-app-container icon-app-container--borderless icon-default-white">
-                  <Image
-                    fallbackSrc={defaultServiceImage}
-                    src={
-                      state.isLoadingSelectedVersion
-                        ? defaultServiceImage
-                        : cosmosPackage.getIcons()["icon-large"]
-                    }
+
+        {!isLoading ? (
+          <div>
+            <div className="container">
+              {this.hasUnresolvedDependency(cosmosPackage) && (
+                <div className="infoBoxWrapper">
+                  <InfoBoxInline
+                    appearance="warning"
```

Comment:
The designers wanted this to be the neutral (grey) InfoBox. You can get this by not passing an `appearance`.